include/swift/SIL/SILInstruction.h

```diff
@@ -4023,19 +4042,35 @@ class ConversionInst : public SingleValueInstruction {
   DEFINE_ABSTRACT_SINGLE_VALUE_INST_BOILERPLATE(ConversionInst)
 };
 
+/// A conversion inst that produces a static OwnershipKind set upon the
```

Comment:
The OwnershipKind doesn't seem to be "static", since there's no way to know from the instruction type. It does seem to constant though.

Reply:
Yes. You are correct that is a better terminology.